packages/samples/headless-ssr/next.config.js

```diff
@@ -1,4 +1,4 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {experimental: {appDir: true}};
```

Comment:
Why is this required, esp with the changes in this PR? Doesn't nextjs 13 default to app dir?

Reply:
Had an error with Next.js, don't think this bit is necessary anymore, forgot to revert it 😅